### PR TITLE
feat: Add node selectors to postgres

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: main.star [hunk 1]
```diff
@@ -32,6 +32,7 @@ def run(
     max_cpu=POSTGRES_MAX_CPU,
     min_memory=POSTGRES_MIN_MEMORY,
     max_memory=POSTGRES_MAX_MEMORY,
+    node_selectors={},
```
Comment (h4ck3rk3y) [inline main.star:35]: we should ideally use `node_selectors=None` and then do

```
if node_selectors == None
    node_selectors = {}
```

https://docs.python-guide.org/writing/gotchas/ - applies to Starlark as well

**Review by h4ck3rk3y — APPROVED** [review]

File: main.star [hunk 2]
```diff
@@ -32,6 +32,7 @@ def run(
...
 ):
```
Comment (h4ck3rk3y) [inline main.star:36]: describe the arg in the doc string please

File: main.star (hunk 1 above)
Comment (barnabasbusa) [reply main.star]: Fixed

File: main.star (hunk 2 above)
Comment (barnabasbusa) [reply main.star:36]: Fixed

**Pull request merged**